Header header
uint8 buttonStart
uint8 buttonRed
uint8 buttonYellow
uint8 buttonBlue
uint8 buttonGreen
float32[4] axes
